eth2deposit/utils/validation.py

```diff
@@ -94,6 +95,20 @@ def validate_deposit(deposit_data_dict: Dict[str, Any], credential: Credential)
     return signed_deposit.hash_tree_root == deposit_message_root
 
 
-def validate_password_strength(password: str) -> None:
+def validate_password_strength(password: str) -> str:
     if len(password) < 8:
-        raise ValidationError(f"The password length should be at least 8. Got {len(password)}.")
+        raise ValidationError(load_text(['msg_password_length']))
+    return password
+
+
+def validate_int_range(num: Any, low: int, high: int) -> int:
+    '''
+    Verifies that `num` is an `int` andlow <= num < high
+    '''
+    try:
+        num_int = int(num)  # Try cast to int
+        assert num_int == float(num)  # Check num is not float
+        assert low <= num_int < high  # Check num in range
+        return num_int
+    except (ValueError, AssertionError):
+        raise ValidationError(load_text(['err_not_positive_integer']))
```

Comment:
It's unclear why it includes `low` but excludes `high`. What do you think about including the given upper bound and adjusting the argument:

```suggestion
def validate_int_range(num: Any, lower_bound: int, upper_bound: int) -> int::
    '''
    Verifies that `num` is an `int` and `lower_bound <= num <= upper_bound`
    '''
    try:
        num_int = int(num)  # Try cast to int
        assert num_int == float(num)  # Check num is not float
        assert lower_bound <= num_int <= upper_bound  # Check num in range
        return num_int
    except (ValueError, AssertionError):
        raise ValidationError(load_text(['err_not_positive_integer']))
```

and call `validate_int_range(num, 0, 2**32 - 1)`.

Reply:
Two reasons:
* It follows python's `range`/indexing convention (eg `x in range(0,8)`, `list[0,8]`)
* `validate_int_range(num, 0, 2**32)` is neater than `validate_int_range(num, 0, 2**32 - 1)`